osmosis-1/osmosis-1.assetlist.json

```diff
@@ -656,6 +656,34 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/med.png"
       },
       "coingecko_id": "medibloc"
+    },
+    {
+      "description": "The staking token of Bostrom",
+      "denom_units": [
+        {
+          "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
+          "exponent": 0,
+          "aliases": [
+            "BOOT"
+          ]
+        },
+        {
+          "denom": "boot",
+          "exponent": 0
```

Comment:
BOOT asset have 0 exponent ? Can you confirme please, thanks

Reply:
Confirmed

Reply:
Hmm, I feel we should not have two `denom_units` entries with the same exponent.  It should just be listed as an alias instead.